frankenstein/consul_client.go

```diff
@@ -0,0 +1,168 @@
+package frankenstein
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
```

Comment:
If you want to be more Prometheus-ecosystem-like, there we're using `github.com/prometheus/common/log`.